.github/workflows/pull-request.yml

```diff
@@ -5,9 +5,21 @@ on:
     types:
       - opened
       - synchronize
-      - reopened
 
 jobs:
+  log:
+    name: 🪵 Log some helpful vars
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          echo "::group::Log some helpful vars"
+          echo "🏷 contains \"staging\" label = ${{ contains(github.event.pull_request.labels.*.name, 'staging') }}"
+          echo "::endgroup:: "
+
   development:
     name: 👷 Development
+    # Development is run for PRs labeled "staging" in deploy-pr-to-staging.yml
+    if: >
+      github.event.action != 'closed' 
+      && !contains(github.event.pull_request.labels.*.name, 'staging')
```

Comment:
Am I reading this incorrectly? The checks in `development.yml` don't run for PRs with staging? Is there are reason we wouldn't want to run these checks? Again, could just be reading the logic wrong.

Reply:
There's a comment above that line which explains the logic, though maybe not well enough!

```yml
# Development is run for PRs labeled "staging" in deploy-pr-to-staging.yml
```

So PRs with the `staging` label run the `development` workflow from `deploy-pr-to-staging.yml`.